multimodal/src/autogluon/multimodal/utils/export.py

```diff
@@ -73,82 +76,71 @@ def dump_model(self, save_path: Optional[str] = None):
 
     def export_onnx(
         self,
-        data: Optional[pd.DataFrame] = None,
-        onnx_path: Optional[str] = None,
+        data: pd.DataFrame,
+        path: Optional[str] = None,
         batch_size: Optional[int] = None,
         verbose: Optional[bool] = False,
         opset_version: Optional[int] = 16,
+        truncate_long_and_double: Optional[bool] = False,
     ):
         """
         Export this predictor's model to ONNX file.
 
         Parameters
         ----------
-        onnx_path
-            The export path of onnx model.
         data
             Raw data used to trace and export the model.
             If this is None, will check if a processed batch is provided.
+        path
+            The export path of onnx model.
         batch_size
             The batch_size of export model's input.
             Normally the batch_size is a dynamic axis, so we could use a small value for faster export.
         verbose
             verbose flag in torch.onnx.export.
         opset_version
             opset_version flag in torch.onnx.export.
+        truncate_long_and_double: bool, default False
```

Comment:
Do we need to do special handing as well for `bf16`?

Reply:
It's handled in the TensorrtExecutionProvider configuration.
```python
"TensorrtExecutionProvider",
{
    "device_id": 0,
    "trt_max_workspace_size": 2147483648,
    "trt_fp16_enable": True,
},
```

Reply:
truncate_long_and_double here just to convert double -> float, int64 -> int32, since TRT wouldn't support int64 and double data types at all.